components/text/hunspell/Makefile

```diff
@@ -41,6 +42,13 @@ CPPFLAGS += -I/usr/include/ncurses
 CFLAGS += -D_XOPEN_SOURCE=600
 CFLAGS += -std=c99
 
+CPPFLAGS +=	-I/usr/include/ncurses
+
+# Check location of ncurses libraries
+ifeq ($(shell test -f /usr/gnu/lib/libncurses.so && echo y),y)
```

Comment:
Of course, but if there is something that can be fixed, it is not by hacking the flags like in this component.